src/main/kotlin/com/cognifide/gradle/aem/common/instance/tail/InstanceLogInfo.kt

```diff
@@ -3,10 +3,9 @@ package com.cognifide.gradle.aem.common.instance.tail
 import com.cognifide.gradle.aem.common.instance.Instance
 import java.time.ZoneId
 
-data class InstanceLogInfo(val name: String, val zoneId: ZoneId) {
```

Comment:
this `val` was wrong here. when instance was initially down, then instead of actual instance zone id, then system default zone id is saved, then timestamps wrongly calculated and in the end wrong dates printed to console